templates/practice_settings/general_list.html

```diff
@@ -3,33 +3,58 @@
+<link rel="stylesheet" type="text/css" href="{php} echo $GLOBALS['assets_static_relative']{/php}/bootstrap-sidebar-0-2-2/dist/css/sidebar.css">
 <link rel="stylesheet" type="text/css" href="{php}echo $GLOBALS['rootdir'] . '/../library/js/fancybox/jquery.fancybox-1.2.6.css';{/php}" media="screen" />
 <script type="text/javascript" src="{php}echo $GLOBALS['assets_static_relative']{/php}/jquery-min-1-3-2/index.js"></script>
+<script type="text/javascript" src="{php}echo $GLOBALS['assets_static_relative']{/php}/bootstrap-3-3-4/dist/js/bootstrap.js"></script>
+<script type="text/javascript" src="{php}echo $GLOBALS['assets_static_relative']{/php}/bootstrap-sidebar-0-2-2/dist/js/sidebar.js"></script>
 <script type="text/javascript" src="{php}echo $GLOBALS['rootdir'] . '/../library/js/common.js';{/php}"></script>
 </head>
-<body class="body_top">
+<body class="body_top" style="padding-top: 35px;">
 
-<div>
-    <b>{xl t='Practice Settings'}</b>
-</div>
+<nav class="navbar navbar-default navbar-fixed-top">
+    <div class="container-fluid">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="#">{php}echo xlt('Practice Settings'){/php}</a>
+        </div>
 
-<div>
-    <div class="small">
-        <a href="{$TOP_ACTION}pharmacy&action=list">{xl t='Pharmacies'}</a> |
-        <a href="{$TOP_ACTION}insurance_company&action=list">{xl t='Insurance Companies'}</a> |
-        <a href="{$TOP_ACTION}insurance_numbers&action=list">{xl t='Insurance Numbers'}</a> |
-        <a href="{$TOP_ACTION}x12_partner&action=list">{xl t='X12 Partners'}</a> |
-        <a href="{$TOP_ACTION}document&action=queue">{xl t='Documents'}</a> |
-        <a href="{$TOP_ACTION}hl7&action=default">{xl t='HL7 Viewer'}</a>
-    </div>
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+            <ul class="nav navbar-nav">
+            </ul>
+            <ul class="nav navbar-nav navbar-right">
+            </ul>
+        </div><!-- /.navbar-collapse -->
+    </div><!-- /.container-fluid -->
+</nav>
 
-    <br/>
-    <div class="section-header">
-        <b>{$ACTION_NAME}</b>
-    </div>
-    <div class="tabContainer">
-        <div class="tab current" style="width:100%">
-            {$display}
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-xs-8 col-sm-2 col-md-2 sidebar sidebar-left sidebar-md-show">
```

Comment:
Getting weird thing happening:
if minimimize screen, then the menu goes away and no menu icon shows up
if continue to minimize the screen, then the menu icon shows up at top right, but it doesn't work (getting a bunch of  js errors stating jquery 1.9.1 at least is needed, so guessing that is causing issues; I would bump it to jquery 3.1.1).